blockchains/canto/info/info.json

```diff
@@ -0,0 +1,28 @@
+{
+    "name": "CANTO",
+    "type": "coin",
+    "symbol": "CANTO",
+    "decimals": 18,
+    "website": "https://canto.io/",
+    "description": "Canto is a Cosmos SDK based Layer-1 blockchain that runs Ethereum Virtual Machine.",
+    "explorer": "https://www.mintscan.io/canto",
+    "status": "active",
+    "rpc_url": "https://canto-rpc.polkachu.com/",
+    "denom": "acanto",
+    "lcd_url": "https://api.canto.nodestake.top/",
+    "hrp": "canto",
+    "links": [
+        {
+            "name": "coinmarketcap",
+            "url": "https://coinmarketcap.com/ru/currencies/canto/"
+        },
+        {
+            "name": "github",
+            "url": "https://github.com/Canto-Network"
+        },
+        {
+            "name": "twitter",
+            "url": "https://twitter.com/cantopublic"
+        }
+    ]
+}
```

Comment:
Similar to https://github.com/trustwallet/assets/blob/master/blockchains/nativeevmos/info/info.json should be NativeCanto